corporate/lib/stripe.py

```diff
@@ -1561,21 +1561,21 @@ def delete_fixed_price_plan(self) -> str:
         customer = self.get_customer()
         assert customer is not None
         current_plan = get_current_plan_by_customer(customer)
-        if current_plan is None:
-            fixed_price_offer = CustomerPlanOffer.objects.filter(
-                customer=customer, status=CustomerPlanOffer.CONFIGURED
+        if current_plan is not None and self.check_plan_tier_is_billable(current_plan.tier):
```

Comment:
This is now the same `if` statement as in `configure_fixed_price_plan` that creates a `CustomerPlan` object and returns.